script.js

```diff
@@ -4,403 +4,294 @@ of the properties make sense to filter on, and some to sort on.*/
   },
   {
-    title: 'The Hunger Games',
-    author: 'Suzanne Collins',
+    title: "The Hunger Games",
+    author: "Suzanne Collins",
     year: 2008,
-    genre: 'Science Fiction',
+    genre: "Science Fiction",
     rating: 4.3,
     description:
       "In a dystopian future, Katniss Everdeen becomes a symbol of rebellion when she volunteers to take her sister's place in the brutal Hunger Games.",
-    image: './books-images/unknown.jpg'
+    image: "./books-images/the_hunger_games_720.jpg",
   },
   {
-    title: 'The Girl with the Dragon Tattoo',
-    author: 'Stieg Larsson',
+    title: "The Girl with the Dragon Tattoo",
+    author: "Stieg Larsson",
     year: 2005,
-    genre: 'Mystery',
+    genre: "Mystery",
     rating: 4.1,
     description:
-      'A gripping mystery novel featuring investigative journalist Mikael Blomkvist and the enigmatic hacker Lisbeth Salander.',
-    image: './books-images/unknown.jpg'
+      "A gripping mystery novel featuring investigative journalist Mikael Blomkvist and the enigmatic hacker Lisbeth Salander.",
+    image: "./books-images/the_girl_with_the_dragon_tattoo_720.jpg",
   },
   {
-    title: 'The Road',
-    author: 'Cormac McCarthy',
+    title: "The Road",
+    author: "Cormac McCarthy",
     year: 2006,
-    genre: 'Dystopian',
+    genre: "Dystopian",
     rating: 4,
     description:
       "Set in a post-apocalyptic world, it follows a father and son's harrowing journey to survive and find safety.",
-    image: './books-images/unknown.jpg'
+    image: "./books-images/the_road_360.jpg",
   },
   {
     title: "The Hitchhiker's Guide to the Galaxy",
-    author: 'Douglas Adams',
+    author: "Douglas Adams",
     year: 1979,
-    genre: 'Science Fiction',
+    genre: "Science Fiction",
     rating: 4.35,
     description:
       "A comedic science fiction series that follows the misadventures of Arthur Dent after Earth's destruction.",
-    image: './books-images/unknown.jpg'
+    image: "./books-images/thgtg_480.jpg",
   },
   {
-    title: 'The Giver',
-    author: 'Lois Lowry',
+    title: "The Giver",
+    author: "Lois Lowry",
     year: 1993,
-    genre: 'Dystopian',
+    genre: "Dystopian",
     rating: 4.12,
     description:
-      'A dystopian novel set in a seemingly perfect society where young Jonas discovers the dark truth beneath the surface.',
-    image: './books-images/unknown.jpg'
+      "A dystopian novel set in a seemingly perfect society where young Jonas discovers the dark truth beneath the surface.",
+    image: "./books-images/unknown.jpg",
+  },
+];
+
+//Declare variables that refer to HTML elements
+const sortNewestBtn = document.getElementById("newestButton");
+const sortOldestBtn = document.getElementById("oldestButton");
+const sortRatingBtn = document.getElementById("ratingButton");
+const randomBtn = document.getElementById("randomButton");
+
+
+//Books is the array. The "for loop" let's us go through the array of books to get the book(s) we need.
```

Comment:
A general comment for the whole code: the comments are very clear and good at explaining what is going on. Well done!